sys/log/full/src/log.c

```diff
@@ -297,12 +297,11 @@ log_find(const char *name)
     struct log *log;
 
     log = NULL;
```

Comment:
This initialization can be removed if you are so inclined.

Reply:
> This initialization can be removed if you are so inclined.
I'm afraid not.
Not that I'm not inclined, it would be wrong.

Reply:
Doesn't `log` get initialized by the while loop?  `log_list_get_next()` has to get called at least once.

Reply:
Oh, I'm sorry I was not clear enough.
You are right that `log` is initialized but it is also used in the call `log_list_get_next(log)` so it's better that at that point it is NULL which means: start from the beginning.
It could easily be missed, and I should explain that in my first reply, I'm sorry for that.

Reply:
Oops, you're right.